content/ngf/overview/technical-specifications.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Technical specifications
+weight: 300
+nd-content-type: reference
+nd-product: FABRIC
+---
+
+This page describes the technical specifications for NGINX Gateway Fabric.
+
+The information included covers version compatibility between NGINX Gateway Fabric and the Gateway API, as well as other NGINX products.
+
+## NGINX Gateway Fabric versions
+
+The following table lists the software versions NGINX Gateway Fabric supports.
+
+| NGINX Gateway Fabric | Gateway API | Kubernetes | NGINX OSS | NGINX Plus | NGINX Agent |
+|----------------------|-------------|------------|-----------|------------|-------------|
+| Edge                 | 1.4.0       | 1.25+      | 1.29.3    | R35        | v3.5.0      |
```

Comment:
I really want there to be a better way to do this. We are updating this table very frequently (it's already out of date here). Keeping this in sync is going to be forgotten, because we already forget to do it sometimes in our own repo.